twa

```diff
@@ -440,6 +440,16 @@ function stage_5_robots_check {
   code=$(fetch_respcode "${domain}/robots.txt")
   if [[ "${code}" -eq 200 ]]; then
     PASS "Site provides robots.txt at: ${domain}/robots.txt"
+    robots=$(fetch -L -s "${domain}/robots.txt")
+
+    # set nocasematch option
+    shopt -s nocasematch
+
+    if [[ $robots =~ .*(admin|cgi-bin).* ]]
```

Comment:
`if [[ cond ]]; then`, for consistency with other conditions.

Reply:
Also, use quotes and explicit-style expansion to prevent unexpected globbing/expansion:

```bash
if [[ "${robots}" =~ .*(admin|cgi-bin).* ]] ; then
  # foo
fi
```

Reply:
Split it into two checks

Shouldn't robots.txt be capitalized because it's at the beginning of the sentence?